[ASM] Expander NGFWv1 (#29831)
demisto/content: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_On_Prem_Remediation_README.md

This playbook adds new block rule(s) to on-prem firewall vendors in order to block internet access for internet exposures.

Conditions:

> - This is currently limited to stand-alone firewalls for PAN-OS.

Reviewer: ```suggestion
This is currently limited to stand-alone firewalls for PAN-OS.
```

## Dependencies

This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

> * PAN-OS - Block Destination Service

Reviewer: ```suggestion
PAN-OS - Block Destination Service
```

Author: this was configured by SDK

### Integrations

> * Panorama

Reviewer: ```suggestion
Panorama
```

### Scripts

This playbook does not use any scripts.

### Commands

> * pan-os-list-rules

Reviewer: ```suggestion
pan-os-list-rules
```

Author: this was configured by SDK

## Playbook Inputs

---

| **Name** | **Description** | **Default Value** | **Required** |
| --- | --- | --- | --- |

> | RuleName | Firewall rule name to lookup to company configuration setting for block rule. |  | Required |

Reviewer: ```suggestion
| RuleName | Firewall rule name to look up in the company configuration setting for block rule. |  | Required |
```
